Source/ui_qt/BootableModel.cpp

```diff
@@ -69,9 +69,10 @@ void BootableModel::SetWidth(int width)
 }
 
 /* start of BootImageItemDelegate */
-BootableCoverQVarient::BootableCoverQVarient(std::string key, std::string title)
+BootableCoverQVarient::BootableCoverQVarient(std::string key, std::string title, std::string path)
```

Comment:
Could we fix the typo in BootableCoverQVarient? I think it should be BootableCoverQVariant.